checking if owftpd is enabled... configure: error: no
make: ***
[/ptx/work/minime/rsc/svn/OSELAS.BSP-phyCORE-trunk/platform-phyCORE-i.MX27/state/owfs.prepare]
Error 1

---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: owfs-2.7p7/configure.ac
===================================================================
--- owfs-2.7p7.orig/configure.ac
+++ owfs-2.7p7/configure.ac
@@ -371,7 +371,7 @@ AC_ARG_ENABLE(owftpd,
 [
     if test ! "$enableval" = "yes" ; then
 	ENABLE_OWFTPD="false"
-	AC_MSG_ERROR([no])
+	AC_MSG_RESULT([no])
     else
 	if test ! "${ENABLE_MT}" = "true" ; then
 	    AC_MSG_ERROR([owftpd needs multithreading])
